client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -13,56 +17,101 @@ const { Search } = Input;
  */
 const RobotOverview = () => {
   const [searchValue, setSearchValue] = useState('');
+  const [userId, setUserId] = useState(1);
+
+  /**
+   * @description Equivalent to ComponentDidMount in class based components
+   */
+  useEffect(() => {
+    getCurrentUserId().then((data) => {
+      setUserId(data.UserId);
+    });
+  }, []);
+
+  /**
+   * @description Updates the current user id on the server side and set the state variable userId to the new value
+   */
+  const changeUserId = (value) => {
+    setCurrentUserId(value).then((data) => {
+      setUserId(data.UserId);
+    });
+  };
 
   const updateSearchValue = (val) => {
     setSearchValue(val);
   };
 
   const createRobotBoxes = (searchValue2) => {
```

Comment:
`searchValue2 `is not so self explaining :D I also dont understand why it's needed here. Is it so to say the "
current user" that is logged in?

Reply:
That function is actually part of the #106 branch and was already documented there. Once we merge this branch into the 106 branch this should be resolved.